EIPS/eip-5485.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: 5485
+title: Legitimacy, Jurisdiction and Sovereignty
+description: An interface for identifying the legitimacy, jurisdiction and sovereignty.
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-5485-interface-for-legitimacy-jurisdiction-and-sovereignty/10425
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-17
+requires: 165, 5247
+---
+
+## Abstract
+Provide a way for compliant smart contracts to declare their legitimacy lineage, jurisdiction they observe, and sovereignty if they choose to not fall onto any jurisdiction.
+
+## Motivation
+Today, smart contract has no standard way to specify their legitimacy lineage, jurisdiction and sovereignty relationship.
+By supporting such a standard, it supports better integrate with today's legal and regulative scencario: 
+1. it supports a regulative body to allow or deny interoperatbility with smart contracts. 
+2. it also allows DAOs to clearely declare "self-sovereignty" by announcing via this interface by saying they do not assert legitimacy from any source
+other then themselves.
+
+A real world example is that ContractA represents an **A company registered in a country**,
+ContractB represents a **The Secretary of State of the country**. The ContractC represents **Supreme Court of the Country**.
+
+Another real example is a contract the declared "self-sovereignty" that doesn't follow any jurisdiction.
+
+This interface supports both cases, providing a way to allow smart contract to determine if they want to allow / prohibit interaction based
+on the sovereignty.
+
+For example A country might want to require any digital money service's all smart contracts to observe their [EIP-5485](./eip-5485.md) jurisdiction before they are allow to operate money in their (real world) legal jurisdiction.
+
+On the other hand, a DAO with strong decentralized ideology might choose to only inter-operate with EOA
+or "self-sovereigned" smart contracts to avoid being affected by any countries.
```

Comment:
```suggestion
On the other hand, a DAO with a strong decentralized ideology might choose to only inter-operate with EOA or "self-sovereign" smart contracts to avoid being affected by any country.
```
Formatting fix required; grammatical fix(es) required.